protocol/contracts/amo/Ramos.sol

```diff
@@ -370,12 +370,12 @@ contract Ramos is IRamos, TempleElevatedAccess, Pausable {
         uint256 feeAmt = quoteTokenAmountIn * rebalanceFees.rebalanceJoinFeeBps / BPS_PRECISION;
         if (feeAmt != 0) {
             emit FeeCollected(address(quoteToken), feeCollector, feeAmt);
-            quoteToken.safeTransfer(feeCollector, feeAmt);
+            tokenVault.borrowQuoteToken(feeAmt, feeCollector);
         }
 
         // Send the remaining quote tokens to the poolHelper
         uint256 joinAmountIn = quoteTokenAmountIn - feeAmt;
-        quoteToken.safeTransfer(address(poolHelper), joinAmountIn);
+        tokenVault.borrowQuoteToken(joinAmountIn, address(poolHelper));
```

Comment:
Also not right...not sure why you're changing the sending of tokens to poolHelper, into borrowing tokens from the vault?

Please put this back